### PR TITLE
Feature/articles

### PR DESCRIPTION
**Pull request by wolf-jozef**
Article Section added with:
- article header,
- article slider (react-slick library: https://www.npmjs.com/package/react-slick),
- article card without rating start element (with mock data from json file).

### CODE REVIEW

File: src/components/PageLayout.tsx [hunk 1]
```diff
@@ -38,13 +38,13 @@ function PageLayout() {
   return (
     <>
       <p>Welcome, {username}</p>
-      {data ? (
+      {/* {data ? (
```
Comment (sacktomasz) [inline src/components/PageLayout.tsx:41]: should we leave this comment for the future or delete it?

Comment (wolf-jozef) [reply src/components/PageLayout.tsx]: gues we can delete it

File: src/components/articleSection/ArticleSlider.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import Button from "./Button";
+import ArrowLeft from "../../assets/arrowLeft.svg";
+import ArrowRight from "../../assets/arrowRight.svg";
+
+interface Props {
+  children?: any;
```
Comment (sacktomasz) [inline src/components/articleSection/ArticleSlider.tsx:10]: for children type we can use "React.ReactNode"

**Review by sacktomasz — APPROVED** [review]

File: src/components/articleSection/ArticleSlider.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+import ArrowRight from "../../assets/arrowRight.svg";
+
+interface Props {
+  children?: React.ReactNode;
+}
+
+const ArticleSlider = ({ children }: Props) => {
+  const [sliderRef, setSliderRef] = useState<any>(null);
+
+  const NextArrow = (props: any) => {
```
Comment (magnowak) [inline src/components/articleSection/ArticleSlider.tsx:16]: would be good to type props so the onClick is an obligatory propr, you can create one and use it both for PrevArrow and NextArrow

File: src/components/articleSection/ArticleSlider.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      {
+        breakpoint: 968,
+        settings: {
+          slidesToShow: 1,
+        },
+      },
+    ],
+  };
+
+  console.log(sliderRef);
```
Comment (magnowak) [inline src/components/articleSection/ArticleSlider.tsx:71]: do we need this?

**Review by magnowak — APPROVED** [review]
two small suggestions

File: src/components/articleSection/ArticleSlider.tsx (hunk 4 above)
Comment (wolf-jozef) [reply src/components/articleSection/ArticleSlider.tsx]: deleted

File: src/components/articleSection/ArticleSlider.tsx (hunk 3 above)
Comment (wolf-jozef) [reply src/components/articleSection/ArticleSlider.tsx]: Prop types added. Not sure how to fix the problem with making onClick obligatory. 
image

**Pull request merged**